.github/workflows/fossa.yml

```
@@ -0,0 +1,18 @@
+name: FOSSA License Scanning
+
+on:
+  - pull_request
+  - push
```

Compatibility with build, terratest workflows.  @k0da , @ytsarev, @somaritane , thoughts? 

```suggestion
on:
  push:
    paths-ignore:
      - '**.md'
      - '**.svg'
      - '**.drawio'
      - '.spelling'
```

---

@kuritka good point